Resources/Prototypes/Entities/Mobs/Player/silicon.yml

```diff
@@ -93,8 +94,8 @@
      2. You may not harm any being, regardless of intent or circumstance.
      3. Your goals are to build, maintain, repair, improve, and power to the best of your abilities, You must never actively work against these goals.
   - type: MovementSpeedModifier
-    baseWalkSpeed : 5
-    baseSprintSpeed : 5
+    baseWalkSpeed : 4
```

Comment:
I don't think this change is needed anymore now that they slow down on hit and are stunnable

Reply:
Reverted